composer.json

```diff
@@ -11,16 +11,16 @@
     ],
     "require": {
         "ext-json": "*",
-        "illuminate/support": "^6.0|^7.0|^8.0",
+        "illuminate/support": "^6.0|^7.0|^8.0|^9.0",
         "facade/ignition-contracts": "^1.0",
```

Comment:
Won't this have the same issue that @codeskills tried to address in his PR?

Reply:
Yes, but I think we should keep the contracts so the solutions work for older versions. I don't think there is a way to support solutions in libraries at the moment without including `ignition` in the project.

Reply:
If `facade/ignition-contracts` doesn't cause issues on L9, I agree that this makes sense. That seems to be the case, looking at the CI build.

If Spatie ends up making `spatie/ignition-contracts` **not** depend on entire ignition, we can try to see if there's a way to support both <L9 and L9

I'll keep this review open

Reply:
Seems like the `ignition-contracts` dependency also makes the package incompatible with PHP 8.1? https://github.com/facade/ignition-contracts/blob/3c921a1cdba35b68a7f0ccffc6dffc1995b18267/composer.json#L20

Reply:
Ah, good catch. I think we'll stay with the old `ignition-contracts` package then, for now